internal/stringify/fmt.go

```diff
@@ -3,6 +3,8 @@ package stringify
 import (
 	"fmt"
 	"reflect"
+
+	"github.com/davecgh/go-spew/spew"
```

Comment:
Any chance we can enable `SortKeys` by default?  Seems like a high quality of life setting.

Reply:
I put it right before the return, but we should did some init